tasks/container.yml

```
@@ -15,6 +15,12 @@
     group: docker
     mode: 0755
 
+- name: Check nim_waku_store_message_db_url for $ sign
+  assert:
+    that: "{{ not nim_waku_store_message_db_url | regex_search('\\$') }}"
+    fail_msg: "Please don't use $ sign: {{ nim_waku_store_message_db_url }}"
```

We don't ask, we tell people what to do.
```suggestion
    fail_msg: 'Do not use $(dollar) sign in passwords: {{ nim_waku_store_message_db_url }}'
```